### PR TITLE
[8.0][FIX][dead_mans_switch_client] Add demo system parameter and some fixes to README.

### LINKED ISSUE #314: [8.0][dead_mans_switch_client] Module crashes runbots
I'm seeing more and more runbots with :x: because of this module. [This seems the offending line](https://github.com/OCA/server-tools/blob/8.0/dead_mans_switch_client/models/dead_mans_switch_client.py#L54). Any clue on how to fix it?

Example runbot: https://runbot.odoo-community.org/runbot/build/3137787

CC @hbrunn.

### PR DESCRIPTION
**Pull request by yajo**
Fix #314.

### CODE REVIEW

**Comment by yajo** [comment]
It seems the fix works. Runbot's issue is now https://github.com/odoo/odoo/issues/10159, but not #314.

cc @hbrunn

File: dead_mans_switch_client/models/dead_mans_switch_client.py [hunk 1]
```diff
@@ -66,3 +67,19 @@ def alive(self):
...
+        name = "dead_mans_switch_client.url"
+        param = conf.get_param(name)
+
+        if not param:
+            url = "{}/dead_mans_switch/alive".format(
+                conf.get_param(
+                    "report.url",
+                    conf.get_param(
+                        "web.base.url",
+                        "http://localhost")))
```
Comment (pedrobaeza) [inline dead_mans_switch_client/models/dead_mans_switch_client.py:84]: Why don't take the value from web.base.url parameter and thus avoid an arbitrary URL that doesn't have to be the correct?

Comment (yajo) [inline dead_mans_switch_client/models/dead_mans_switch_client.py:84]: It's what I'm doing here. The `http://localhost` part is only for when `web.base.url` is not available.

Comment (pedrobaeza) [inline dead_mans_switch_client/models/dead_mans_switch_client.py:84]: Oh, sorry, very late review...

**Comment by pedrobaeza** [comment]
:+1:

**Comment by gurneyalex** [comment]
:+1:

**Comment by hbrunn** [comment]
:+1:

**Pull request merged**